custom.go

```diff
@@ -0,0 +1,48 @@
+package auth0
```

Comment:
not married to having this type inside `package auth0`, opening to suggestions where to move it (`internal/custom`)?

Reply:
moved this back to `management/users`